src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Properties.jl

```diff
@@ -654,6 +650,55 @@ end
 is_smooth(X::AbsSpec{<:Field, <:MPolyRing}) = true
 is_smooth(X::AbsSpec{<:Field, <:MPolyLocRing}) = true
 
-@attr function is_integral(X::AbsSpec)
-  return Oscar._is_integral_domain(OO(X))
+###################################################################
+# Irreducibility and Integrality                                  #
+#    integral iff (irreducible && reduced)                        #
+#    integral = OO(X) is integral domain                          #
+#    irreducible = nilradical of OO(X) is prime                   #
+###################################################################
+@doc Markdown.doc"""
+   is_irreducible(X::AbsSpec)
+
+Return the boolean value whether an affine scheme `X` is irreducible.
```

Comment:
This is rather subjective but "the boolean value" seems a bit redundant. Also, we are talking about a concrete object. So I'd use this:
```suggestion
Return whether the affine scheme `X` is irreducible.
```
That said, this is merely a suggestion, and also not about anything important. So I am totally fine with merging this as-is. We can at some future point decide if we want to have someone go over all our docstrings to make this kind of thing a bit more consistent across various parts of the documentation.